freebasics/forms.py

```diff
@@ -1,6 +1,7 @@
 from django import forms
 from django.contrib.auth.models import User
 from django.utils.translation import ugettext_lazy as _
+from phonenumber_field.formfields import PhoneNumberField
 
 
 class RegistrationForm(forms.Form):
```

Comment:
This needs to inherit from `molo.profiles` similar to what we did in [molo.gem](https://github.com/praekelt/molo-gem/blob/develop/gem/forms.py#L7-L12)